src/pages/leeszaal/[slug].tsx

```diff
@@ -21,13 +56,21 @@ const ReadingRoomPage: NextPage = () => {
 				<Navigation.Right />
 			</Navigation>
 
-			<div style={{ padding: '3.2rem' }}>
-				<Placeholder
-					img="/images/lightbulb.svg"
-					title="Start je zoektocht!"
-					description="Zoek op trefwoorden, jaartallen, aanbieders… en start je research."
-				/>
-			</div>
+			<section className="u-bg-black">
+				<div className="l-container">
+					<ScrollableTabs tabs={tabs} onClick={onTabClick} />
+				</div>
+			</section>
```

Comment:
Maakt dit geen deel uit van de contextual nav? Wat is het scroll behaviour van die tabs? Blijven ze zichtbaar bij het scrollen of scrollen ze onder de nav?

Reply:
Nee, wordt ook nergens zo gedefinieerd als onderdeel van de contextuele nav. Dat zijn in prncipe high level filters voor die specifieke pagina. Moesten de tabs andere views triggeren zou je het anders kunnen zien natuurlijk.
Het scroll behaviour van de tabs wordt pas zichtbaar als er overflow is, dit gaat enkel voorvallen op kleinere schermen.